app/experimenter/experiments/constants.py

```diff
@@ -177,7 +178,11 @@ class Type(models.TextChoices):
         Status.DRAFT: (Status.PREVIEW,),
         Status.PREVIEW: (Status.DRAFT,),
     }
-    STATUS_ALLOWS_UPDATE = (Status.DRAFT,)
+
+    STATUS_ALLOWS_UPDATE = (
+        Status.DRAFT,
+        Status.LIVE,
+    )
```

Comment:
Hrmmm interesting.  We want to allow edits for live rollouts but not live experiments, I wonder if we'll have to expand this logic to be more specific?  Can figure it out later as we get more into the weeds with it, just something to think about.